src/sqs/error-handlers.ts

```diff
@@ -81,14 +89,19 @@ const maybeHandleNonRetryableResponseCode = <MessagePayload extends BaseMessageP
 	const status: number | undefined = error["status"] as number | undefined;
 	if (status && UNRETRYABLE_STATUS_CODES.includes(status)) {
 		context.log.warn({ err: error }, `Received error with ${status} status. Unretryable. Discarding the message`);
-		return { retryable: false, isFailure: false };
+		return {
```

Comment:
/nit and existing and dont do, next time we are in the neighbourhood lol

but we repeat return {
			isFailure: false,
			statusCode: cause.status,
			source: "github",
			errorName: cause.constructor.name
		}; a few times, would be worth just declaring it up top then returning